pkg/apis/v1alpha1/vpc_types.go

```diff
@@ -38,23 +38,12 @@ type VPCStatus struct {
 	Conditions []Condition `json:"conditions"`
 	// NSX VPC Policy API resource path.
 	NSXResourcePath string `json:"nsxResourcePath"`
-	// Default SNAT IP for private Subnets
-	DefaultSNATIP string            `json:"defaultSNATIP"`
-	CIDRsUsage    VPCCIDRsUsageInfo `json:"cidrsUsage"`
-}
-
-type VPCCIDRsUsageInfo struct {
-	PublicCIDRsUsage  CIDRsUsageInfo `json:"publicCIDRsUsage"`
-	PrivateCIDRsUsage CIDRsUsageInfo `json:"privateCIDRsUsage"`
-}
-
-type CIDRsUsageInfo struct {
-	// Allocated IPs in the IP Block (including IPs used by other VPCs).
-	Allocated int `json:"allocated"`
-	// Used IP for the VPC in the IP Block (IPs used only in current VPC).
-	Used int `json:"used"`
-	// Total IP number in the IP Block.
-	Total int `json:"total"`
+	// Default SNAT IP for private Subnets.
+	DefaultSNATIP string `json:"defaultSNATIP"`
+	// NSX PolicyPath for LB Subnet.
```

Comment:
Let us confirm with AVI team they really need NSX Operator to create the Subnet and need the path or not.